components/eventing-controller/pkg/handlers/beb.go

```diff
@@ -249,7 +283,7 @@ func (b *Beb) deleteSubscription(name string) error {
 		return fmt.Errorf("delete subscription failed: %v", err)
 	}
 	if resp.StatusCode != http.StatusNoContent && resp.StatusCode != http.StatusNotFound {
-		return fmt.Errorf("delete subscription failed: %v; %v", resp.StatusCode, resp.Message)
+		return fmt.Errorf("delete subscription failed: %w; %v", errors.New(strconv.Itoa(resp.StatusCode)), resp.Message)
```

Comment:
Creating 2 new error types is unnecessary IMO.

Reply:
Yes, but this is conform with the new "errors" package added by golang.